File: 0_sliding_window/numSubarraysWithSum/brute.js
/**
 * @param {number[]} nums
 * @param {number} goal
 * @return {number}
 */
const getSum = (nums, i, j) => {
  let sum = 0;
  for (let x = i; x <= j; x++) {
    sum += nums[x];
  }
  return sum;
};
var numSubarraysWithSum = function (nums, goal) {
  let n = nums.length;
  let cnt = 0;
  for (let i = 0; i < n; i++) {
    for (let j = i; j < n; j++) {
      if (getSum(nums, i, j) == goal) {
        cnt++;
      }
    }
  }
  return cnt;
};

File: 0_array/question/numberOfInversions/brute.cpp

#include <iostream>
#include <map>
#include <vector>
using namespace std;
class Solution {
   public:
    long long int numberOfInversions(vector<int> nums) {
        int cnt = 0;
        for (int i = 0; i < nums.size(); i++) {
            for (int j = i; j < nums.size(); j++) {
                if (nums[i] > nums[j]) {
                    cnt++;
                }
            }
        }
        return cnt;
    }
};